### ISSUE TITLE
Error (AttributeError: 'SpaceView3D' object has no attribute 'edit_tree') when Blender is launched in command line

### ISSUE THREAD

**Issue by xan2622** [Trantor2098/hot_node#5]
Hi. 
I have installed Hot Node 0.7.4 (from the [Blender Extensions Platform](https://extensions.blender.org/add-ons/hot-node/)) on Blender 4.2.3 but when I launch Blender in command line, here is the message that I see: 

```
Traceback (most recent call last):
  File "C:\Users\xan2622\AppData\Roaming\Blender Foundation\Blender\4.2\extensions\blender_org\hot_node\operators.py", line 715, in poll
    return isinstance(context.space_data, bpy.types.SpaceProperties) or context.space_data.edit_tree is not None
                                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'SpaceView3D' object has no attribute 'edit_tree'
```

**Comment by Trantor2098** [comment]
Hi xan2622, I cannot reproduce this error info when launch blender via command line. It seems that the error is about View3D layer, but Hot Node is at the NodeEditor Layer. Does the addon works fine in the Node Editor?

But indeed there is a risk of this error, I will update the addon later to avoid this.

**Comment by xan2622** [comment]
> Does the addon works fine in the Node Editor?

As far as I can tell, it works fine.

**Comment by Trantor2098** [comment]
OK, fix will come and i will close this issue for now. reopen if any issues appeard

**Comment by Trantor2098** [comment]
This may been solved in the new released version 0.7.5,  im not sure but maybe.
